### DISCUSSION TITLE
How do you pass args to just the "target" of filter?

### DISCUSSION THREAD

**Discussion by NullVoxPopuli** [vercel/turborepo#1148] [Help] [answered]
For example, say your command is
```
turbo run build --filter="limber" -- --environment=development
```

I _want_ to only pass `--environment=development" to the "limber" project

however, what is happening now is that it's passed to _all_ dependencies:
```
❯ turbo run build --filter="limber" -- --environment=development
• Packages in scope: limber
• Running build in 1 packages
@nullvoxpopuli/limber-styles:build: cache miss, executing abce1fbd45d2248f
@nullvoxpopuli/horizon-theme:build: cache miss, executing 5fb5492b4ad2ad75
@nullvoxpopuli/horizon-theme:build:  ERROR   ERROR  Unknown option: 'environment'
@nullvoxpopuli/horizon-theme:build: For help, run: pnpm help run
@nullvoxpopuli/horizon-theme:build: Error: command finished with error: command (packages/horizon-theme) pnpm run build --environment=development exited (1)
command (packages/horizon-theme) pnpm run build --environment=development exited (1)
```

I'm using `turbo@1.2.5`

The reason I need to know how to do this is because each project can use a different build tool (ESBuild, rollup, SWC, you name it) -- and not everything supports the same flags

**Comment by dante01yoon** [comment]
I'm facing this issue too. How can I handle this?

**Comment by mehulkar** [comment, marked as answer]
This is a good callout and unfortunately not possible. Feel free to file an feature request for this along with any ideas for what the syntax would look like.
